_includes/metadata.html

```diff
@@ -18,8 +18,10 @@
     {%- else -%}
         {%- capture title -%}{{ site.title }}{%- endcapture -%}
     {%- endif -%}
-    {%- if page.title != "blank" and page.url != "/" -%}
-        {{ title }} | {{ page.title }}
+    {% if page.url == '/blog/index.html' %}
+        {{ site.blog_nav_title }} | {{ title }}
+    {%- elsif page.title != "blank" and page.url != "/" -%}
+        {{ page.title }} | {{ title }}
```

Comment:
is flipping the order from `{{ title }} | {{ page.title }}` to `{{ page.title }} | {{ title }}` intentional? (I don't have a particular preference, just wondering if your decision was motivated by something.)

Reply:
makes sense. thanks for researching this!